static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -56,6 +57,10 @@ type Props = {
    * Tag name for the outer wrap, defaults to `div`
    */
   renderWrapAs?: React.ElementType;
+  /**
+   * Affects the size of the trigger button and menu items.
+   */
+  size?: keyof Theme['form'];
```

Comment:
nit: since you're doing this a few times, I think it might read better if you exported a named type from the Theme file, like `FormElementSize` or something.

Reply:
Ahh good point! Did so in https://github.com/getsentry/sentry/pull/37206/commits/b713ad3574d4e0900e327d9122771438f33a7bc9